cmd/init.go

```diff
@@ -0,0 +1,163 @@
+package cmd
+
+import (
+	"fmt"
+	"path"
+	"text/template"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/fsext"
+)
+
+const defaultNewScriptName = "script.js"
+
+//nolint:gochecknoglobals
+var defaultNewScriptTemplate = template.Must(template.New("init").Parse(`import http from 'k6/http';
```

Comment:
Have you considered moving this into a separate file(s)?

It also could benefit us by introducing one more flag that can control the template and just running the 

```
k6 init
```
will produce a simple example
```javascript
import http from 'k6/http';
import { sleep } from 'k6';

export default function() {
  http.get('https://test.k6.io');

  sleep(1);
}

```

by running the

```
k6 init --template cloud
```
Produces the script with the cloud options set up, browser one produces the script with some browser usage, and so on...

My rationale is that this could produce a simpler script that could be more readable shrug:

Reply:
We can definitely iterate on the script content, but I'd like to do it outside of the scope of this PR. I did consider adding a flag to skip the comments, similar to `vagrant init --minimal`, but then decided to not to include to keep the PR focused on introducing the command. My main target was to provide users who are new to k6 with some directions and structure to author their first script. I believe this also simplifies initializing a new script file for experienced users, since the cognitive load required to remove a comment is lower than to write a few lines of valid JS code :) But, again, I think this is a good idea that is fairly simple to implement, so I'm up for contributing in separately.